test/e2e_tests/placement.go

```diff
@@ -71,4 +72,28 @@ var _ = Describe("Validate the placement controller", func() {
 			return assertionCapp.Annotations[testconsts.AnnotationKeyHasPlacement] == assertionCapp.Spec.Site
 		}, testconsts.Timeout, testconsts.Interval).Should(BeTrue(), "Should fetch capp.")
 	})
+
+	It("Should update the score of the AddOnPlacementScore", func() {
+		baseCapp := mock.CreateBaseCapp()
+		baseCapp.Spec.Site = testconsts.Cluster2
+		desiredCapp := utilst.CreateCapp(k8sClient, baseCapp)
+
+		By("Checking the score of the AddOnPlacementScore in chosen site")
+		addonPlacementScore := &clusterv1alpha1.AddOnPlacementScore{}
+		Eventually(func() int32 {
+			addonPlacementScore = utilst.GetAddOnPlacementScore(k8sClient, testconsts.AddOnPlacementScoresName, testconsts.Cluster2)
+			return addonPlacementScore.Status.Scores[0].Value
+		}, testconsts.Timeout, testconsts.Interval).ShouldNot(BeNil(), "Should have a score")
+		score := addonPlacementScore.Status.Scores[0].Value
+
+		By("Checking unique creation of Capp")
```

Comment:
dont know why this check is needed

Reply:
It serves to make sure the Capp is created